src/utils/mixpanel.ts

```diff
@@ -1,92 +0,0 @@
-import mixpanel from 'mixpanel-browser';
```

Comment:
this file was migrated to a combination of `contexts/analytics/authUtil` and `AnalyticsContext`